libPhoneNumberTests/NBPhoneNumberOfflineGeocoderTest.m

```diff
@@ -0,0 +1,387 @@
+//
+//  NBPhoneNumberOfflineGeocoderTest.m
+//  libPhoneNumberGeocodingTests
+//
+//  Created by Rastaar Haghi on 6/23/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import "NBPhoneNumber.h"
+#import "NBPhoneNumberOfflineGeocoder.h"
+#import "NBPhoneNumberUtil.h"
+@interface NBPhoneNumberOfflineGeocoderTest : XCTestCase
+
+@property(nonatomic, strong) NBPhoneNumberOfflineGeocoder *geocoder;
+
+@property(nonatomic, readonly, copy) NBPhoneNumber *KO_NUMBER1;
+@property(nonatomic, readonly, copy) NBPhoneNumber *KO_NUMBER2;
+@property(nonatomic, readonly, copy) NBPhoneNumber *KO_NUMBER3;
+@property(nonatomic, readonly, copy) NBPhoneNumber *KO_INVALID_NUMBER;
+@property(nonatomic, readonly, copy) NBPhoneNumber *KO_MOBILE;
+@property(nonatomic, readonly, copy) NBPhoneNumber *US_NUMBER1;
+@property(nonatomic, readonly, copy) NBPhoneNumber *US_NUMBER2;
+@property(nonatomic, readonly, copy) NBPhoneNumber *US_NUMBER3;
+@property(nonatomic, readonly, copy) NBPhoneNumber *US_NUMBER4;
+@property(nonatomic, readonly, copy) NBPhoneNumber *US_INVALID_NUMBER;
+@property(nonatomic, readonly, copy) NBPhoneNumber *NANPA_TOLL_FREE;
+@property(nonatomic, readonly, copy) NBPhoneNumber *BS_NUMBER1;
+@property(nonatomic, readonly, copy) NBPhoneNumber *AU_NUMBER;
+@property(nonatomic, readonly, copy) NBPhoneNumber *GERMAN_NUMBER;
+@property(nonatomic, readonly, copy) NBPhoneNumber *ITALIAN_NUMBER;
+@property(nonatomic, readonly, copy) NBPhoneNumber *AR_MOBILE_NUMBER;
+@property(nonatomic, readonly, copy) NBPhoneNumber *NUMBER_WITH_INVALID_COUNTRY_CODE;
+@property(nonatomic, readonly, copy) NBPhoneNumber *INTERNATIONAL_TOLL_FREE;
```

Comment:
These aren't really constants so their names should just be normal lowerCamelCase

Reply:
Done.